net-p2p/mktorrent/mktorrent-1.1_p20180502.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit toolchain-funcs
+
+DESCRIPTION="Simple command line utility to create BitTorrent metainfo files"
+HOMEPAGE="https://github.com/Rudde/mktorrent"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Rudde/mktorrent.git"
+else
+	COMMIT_ID="96090fb175f3cef17ae2499e98c2868363106927"
+	SRC_URI="https://github.com/Rudde/${PN}/archive/${COMMIT_ID}.tar.gz -> ${PN}-${PVR}.tar.gz"
```

Comment:
Please name it `${PN}-${COMMIT_ID}.tar.gz` to match the directory inside it. Also, including revision in distfile name would be wrong. Also, `${PN}-${PVR}` == `${PF}`.